flow-defs/src/utils.js.flow

```diff
@@ -17,6 +17,7 @@ export type ShareOptions =
       fileName: string,
       text?: string,
     };
+export type RemoteConfig = {result: {[s: string]: string}};
```

Comment:
I think it makes sense to expose result: {...} so that if we need to expose any first level known props in the future they won't be mixed with the abritrary strings